axol/config.py

```diff
@@ -51,67 +52,61 @@
+        ['CAACAgQAAx0CQgMQgQABAYl4XvUIi9fLohxAaOtNNNMAAWDCmYO-AAKZAAMVdukH9Btqq0RjzZgaBA']],
     [['D: D: D: D: D: D: D: D: D: D:'], [], 0, [], ['CAADAgADHQADjW7LE5T5heR8tu6uAg']],
-    [[], ['CAADAgADHQADO9HBD8DTsJ6PcoXXAg'], rinneko_chat, ['О, РИНЕЙКА.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
+    [[], ['AgADHQADO9HBDw'], rinneko_chat, ['О, РИНЕЙКА.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
                                                             'О, РИННЕКО.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
                                                             'О, РИНЖЕНЕР.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
                                                             'О, КОТЕЙКА.\nЗАКИДЫВАТЬ МОЛНИЯМИ!'], []],
-    [[], ['CAADAgADjQADP_vRD6t7L9YKnP1eAg'], 44989459,
-     ["https://pp.userapi.com/c639627/v639627599/5b009/flPJ3T4a_o8.jpg"], ['CAADAgADoQADg8cnEW-_9xNR4RKLAg']],
     [[':D:'], [], 0, [], ['CAADAgADGgADjW7LE0g-GMhzIUFiAg']],
     [['РИННЕКО', 'РИН-НЕ-КО', 'РИННЕКО!', 'РИН-НЕ-КО!'], [], rinneko_chat, [], ['CAADAgADxAADP_vRD-wohLnM0GMRAg']],
-    [[], ['CAADAgADKwQAAuce7AWo_pyLoNBvLgI'], 196345058, ['О, НАТАЛИЯ НАЧАЛАСЬ'], []],
+    [[], ['AgADKwQAAuce7AU'], nata_chat, ['О, НАТАЛИЯ НАЧАЛАСЬ'], []],
     [["1Л"], [], 0, ["2Л"], []],
     [["/ТИХО РЖЁТЬ/"], [], osa_chat, ["ГРОМЧЕ!"], []],
     [["/АРЁТ/"], [], osa_chat, [], ['CAADAgADDQADdUTxCP-onsxKIsJqAg']],
     [["/start"], [], 0, ["ТЫ КТО? И КАК ОБО МНЕ УЗНАЛ?"], []],
-    [["ДА", "ДА."],[],0,["ША"],[]],
 
-    [[], ['CAADAgADpgEAAmDrzgNSIT8rlE3K0AI', 'CAADAgADVQADsjRGHmcXPWcGP2BfAg'], 0, [], [], "task_rework"],
-    [fail_phrase, ['CAADAgADFAADjW7LE9E1TbwjuTk7Ag'], 0, [], [], "task_fail"],
-    [approve_phrase, ['CAADAgAD4QADNnYgDo3M52SK-Y3iAg'], 0, [], [], "task_complete"],
+    [[], ['AgADpgEAAmDrzgM', 'AgADVQADsjRGHg'], 0, [], [], "task_rework"],
+    [fail_phrase, ['AgADFAADjW7LEw'], 0, [], [], "task_fail"],
+    [approve_phrase, ['AgAD4QADNnYgDg'], 0, [], [], "task_complete"],
     [['КЛАЦ!'], [], 0, [], [], "task_extra"],
-    [["НАТАЛЬЯ?"], ['CAADAgADIwADP_vRD4CXRh4oYhhXAg', "CAADAgADHwADP_vRD3g8MWNndtchAg"], 0,
+    [["НАТАЛЬЯ?"], ['AgADHwADP_vRDw'], 0,
         ['НАТАЛИРУЙ', "@Abi_Abigale", "ВЫ ЧО ТУТ, В ГОДВИЛЛЬ ИГРАЕТЕ?"], ["CAADAgADJAADP_vRDykJokH6fiyLAg"], "natalka"],
     [['ВЫГОНИ ПРОЧЬ ЭТИХ НАДОЕДЛИВЫХ БОТОВ!'], [], citrus_chat, [], [], "kick_bots"],
 
     [['АНТИКЛАЦ!'], [], 0, [], [], "anti_task"],
-    #[[], ['CAADAgADnAADg8cnESB0jVNnakYoAg'], 0, [], [], "whois"],
+    #[[], ['AgADnAADg8cnEQ'], 0, [], [], "whois"],
     [["ПЕРЕНОСИМ"], [], 0, [], [], "stop_natalka"],
-    [[], ['CAADAgADLgADsjRGHsGbRuS2PA26Ag'], 0, [], [], "kick_citrus"],
-    [[], ['CAADAgADKAADO9HBD1SvkhGZJT_BAg', 'CAADAgADVwADO9HBD35E69ryx07IAg'], 0, [], [], "kick_rels"],
-    [[], ['CAADAgADLwADsjRGHvSXP4ESf3tIAg'], 0, [], [], "kick_misha"],
+    [[], ['AgADLgADsjRGHg'], 0, [], [], "kick_citrus"],
+    [[], ['AgADKAADO9HBDw', 'CAADAgADVwADO9HBD35E69ryx07IAg'], 0, [], [], "kick_rels"],
```

Comment:
Please, fix second sticker or just remove it

Reply:
Done